.github/workflows/deploy-template.yml

```diff
@@ -81,6 +81,9 @@ jobs:
           for env in ${envs[*]};
           do
             yq e -i '(.vets-api.image.tag) |= "${{ github.sha }}"' $env/values.yaml
+            yq e -i '(."vets-api".web.envSecrets[] | select(.name=="DD_VERSION").value) |= "${{ github.sha }}"' $env/values.yaml
+            yq e -i '(."vets-api".web.dbMigrate.envSecrets[] | select(.name=="DD_VERSION").value) |= "${{ github.sha }}"' $env/values.yaml
+            yq e -i '(."vets-api".worker.envSecrets[] | select(.name=="DD_VERSION").value) |= "${{ github.sha }}"' $env/values.yaml
```

Comment:
What's the reason for having `"vets-api"` in quotes on these three lines?

Reply:
It was necessary because of the hyphen when I was testing locally, it would probably be good to add it to the existing line as well. Maybe it's due to a yq version difference?

Reply:
possibly. I bet both versions work.

Reply:
The `jq` manual says that [keys with special chars should be quoted](https://jqlang.github.io/jq/manual/#object-identifier-index) - I went ahead and added it to the first line for consistency.

Reply:
ok sounds good!